app/src/main/java/com/scottyab/challenge/presentation/ui/common/TopBars.kt

```diff
@@ -0,0 +1,31 @@
+package com.scottyab.challenge.presentation.ui.common
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowBack
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import com.scottyab.challenge.R
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun DefaultTopBar(
```

Comment:
Example of basic screen building blocks that I'd look to create as the compose UI implementation grows to encourage consistency across the UI